python/raft-ann-bench/src/raft-ann-bench/data_export/__main__.py

```diff
@@ -62,6 +92,27 @@ def convert_json_to_csv_search(dataset, dataset_path):
                 "qps": df["items_per_second"],
             }
         )
+        for name in df:
+            if name not in skip_search_cols:
+                write[name] = df[name]
+
+        if os.path.exists(build_file):
```

Comment:
Can we do an else here and print a warning when the corresponding build file could not be found?